recipe/meta.yaml

```diff
@@ -10,19 +10,22 @@ source:
   sha256: 5554276e18bdcddf387a08c2dd23f9da310c6598905df6a2a244516c22ded9aa
 
 build:
-  number: 0
+  number: 1
 
 outputs:
   - name: {{ name }}
     script: build_linux.sh      # [not win]
     script: build_win.bat       # [win]
     requirements:
       build:
-        - {{ compiler('c') }}
+        - {{ compiler('c') }}              # [not win]
+        - {{ stdlib("c") }}                # [not win]
+        - make                             # [not win]
+        - {{ compiler('m2w64_c') }}        # [win]
+        - {{ stdlib("m2w64_c") }}          # [win]
```

Comment:
Might not fix this particular error, but I think these two lines are a problem on Windows.  I tested locally and they can be removed without triggering the warning from the recipe linter.  Without them I think Windows defaults to clang, which was working in earlier recipes.

Reply:
After editing, _might_ have to rerender.